utilities/cloudharness_utilities/deployment-configuration/helm/files/db-mongo-backup.sh

```diff
@@ -0,0 +1,35 @@
+/bin/bash << 'EOF'
```

Comment:
may be add
```
#!/bin/bash

set -e
set -u
```

Reply:
or do you like here documents more

Reply:
I used here documents to have multiline container args with proper interpretation of conditional statements, but I noticed it's not necessary anymore, since I extracted the script to a separate file and used `.Files.Get `

Your suggestion is nice, I applied it!